hw/ip/otbn/dv/uvm/otbn_sim_cfg.hjson

```diff
@@ -53,20 +55,34 @@ name:
   uvm_test: otbn_base_test
   uvm_test_seq: otbn_base_vseq
 
-  // The directory where ELF binaries should be found.
-  //
-  // TODO: This needs deciding upon. Eventually, it might be nice to allow this
-  //       to be overridden on the command line ("I've got a bunch of ELF files
-  //       that stress my new feature; please run the test suite with them")
+  // A build mode that sets up the toolchain. Run meson via
+  // meson_init.sh to generate the .env file that describes our toolchain.
+  build_modes: [
+    {
+      name: otbn_build_env_mode
+      pre_build_cmds: ["cd {proj_root} && BUILD_ROOT={build_dir} ./meson_init.sh"]
+    }
```

Comment:
I suppose the run mode below needs to be enabled with this build_mode:
```suggestion
      en_run_modes: ["build_otbn_rig_binaries_mode"]
    }
```

Reply:
I'm not sure that works. I don't want to run the random instruction generator for e.g. the automated CSR tests.

Reply:
Ohh, I get the problem. Before your latest force-push, you had enabled `otbn_build_env_mode` by default (bare in the HJson as opposed to within the test specifications), which affected CSR and all other tests that do not actually need it. By ditching that build mode, this problem is no longer there.